src/app/views/discover/games/list/list.ts

```diff
@@ -106,98 +132,148 @@ export default class RouteDiscoverGamesList extends BaseRouteComponent {
 
-				case 'new':
-					return this.$gettextInterpolate('New %{tag} games', { tag: this.tag });
+		const title = this.$gettextInterpolate('%{ listOf } %{ free } %{ gamesType }', {
+			listOf: this.displayListOf,
+			free: onlyFree ? this.$gettext('free') : '',
+			gamesType: this.displayGamesType,
+		});
 
-				default:
-					return this.$gettextInterpolate('Browsing %{tag} games', { tag: this.tag });
-			}
-		}
+		// Remove duplicate whitespaces (formed when a replacement ends up being an empty string)
+		return title.replace(/\s+/g, ' ');
+	}
 
+	get displayListOf() {
 		switch (this.section) {
 			case 'featured':
-				return this.$gettext('Featured games');
-
+				return this.$gettext('Featured');
 			case 'best':
-				return this.$gettext('Best games (top-rated)');
-
+				return this.$gettext('Best');
 			case 'new':
-				return this.$gettext('New games');
+				return this.$gettext('Newest');
+			default:
+				return this.$gettext('Browse');
+		}
+	}
+
+	get displayGamesType() {
+		if (!this.tag) {
+			return this.$gettext('games');
+		}
 
+		switch (this.tag) {
+			case 'horror':
+				return this.$gettext('scary horror games');
+			case 'fangame':
+				return this.$gettext('fan games');
+			case 'fnaf':
+				return this.$gettext(`Five Nights at Freddy's games`);
+			case 'scifi':
+				return this.$gettext('science fiction games');
+			case 'retro':
+				return this.$gettext('nostalgic and retro games');
+			case 'pointnclick':
+				return this.$gettext('point and click games');
+			case 'altgame':
+				return this.$gettext('alt games');
+			case 'roguelike':
+				return this.$gettext('rogue like games');
 			default:
-				return this.$gettext('Browsing games');
+				return this.$gettextInterpolate(`%{ category } games`, {
+					category: this.tag,
+				});
 		}
 	}
 
 	/**
 	 * This function doesn't translate, but it's only used for meta tags
 	 * currently.
 	 */
-	get routeMeta() {
-		let descriptiveTag = `games`;
-		if (this.tag === 'fangame') {
-			descriptiveTag = `fangames`;
-		} else if (this.tag === 'scifi') {
-			descriptiveTag = `sci-fi games`;
-		} else if (this.tag === 'rpg') {
-			descriptiveTag = `rpgs`;
-		} else if (this.tag === 'rpg') {
-			descriptiveTag = `altgames`;
-		} else if (this.tag === 'fnaf') {
-			descriptiveTag = `five nights at freddy's fangames`;
-		} else if (this.tag) {
-			descriptiveTag = `${this.tag} games`;
-		}
-
+	get listDescription() {
 		if (!this.section) {
-			return `Browse the freshest ${descriptiveTag} on Game Jolt. They're almost too hot!`;
+			return this.$gettextInterpolate(`Browse the trending %{ gamesType } on Game Jolt.`, {
+				gamesType: this.displayGamesType,
+			});
 		} else if (this.section === 'new') {
-			return `Browse new ${descriptiveTag} on Game Jolt. Good or bad, you decide!`;
+			return this.$gettextInterpolate(
+				// tslint:disable-next-line:max-line-length
+				`Browse new %{ gamesType } on Game Jolt. You can find anything here from hidden gems to doodoo land mines!`,
```

Comment:
I think for all these, you should remove the extra flavor text like, "You can find anything..." I think it's distracting from them seeing the game titles, which are more important.

Reply:
The extra games only show for description meta tags, not for users - for them, I wanted it to be a bit more verbose to set the expectations haha.

Reply:
Maybe we can special case the "best" section to show the game lists without the flavor text since it'll probably be games people know?